envs/fishhook/fishhook.post-deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!env bash
+set -o pipefail
+set -e # Exit on error
+
+R --vanilla -q -e 'devtools::install_github("mskilab/gUtils", dependencies = FALSE, upgrade = "never")' > /home/sgillis/fishhook_deploy.txt
```

Comment:
Does this path to home dirs needs to be updated?

Reply:
It needs to be removed entirely actually. Thanks for catching that!